client/src/pages/admin/tasks.tsx

```diff
@@ -63,6 +64,21 @@ function Page(props: Props) {
 
   const handleModalSubmit = useCallback(
     async (values: any) => {
+      const checkCriteria = () => {
+        return dataCriteria.every(item => {
+          if (item.type !== TaskType.Title) {
+            return item.max !== 0;
+          }
+          return;
+        });
+      };
+
+      const isVerified = dataCriteria.length ? checkCriteria() : true;
```

Comment:
Minor: I would shortened it a bit  
```suggestion
      const isVerified = !dataCriteria.length || checkCriteria();
```

Reply:
i think i disagree with the suggested solution as it is harder to understand the condition. the first is clear and simple